mvkGitRevDerived.h.in

```diff
@@ -0,0 +1,2 @@
+// Auto-generated by MoltenVK
+static const char* mvkRevString = "@MVK_GIT_REV@";
```

Comment:
I'd like to avoid cluttering the top level directory if possible. Can we find a different home for this file? Maybe under `Templates`?

Reply:
I'll move it there.